task/06-conditions-n-loops-tasks.js

```diff
@@ -226,10 +268,9 @@ function reverseString(str) {
  *   34143 => 34143
  */
 function reverseInteger(num) {
-    throw new Error('Not implemented');
+    return reverseString(num);
```

Comment:
Using existed functions is ok in real projects, but here, the previous function might be changed for some reason.
At least reverseString returns string type. 
The number is expected as a result of reverseInteger (here it's converted automatically to number when compared to number in test)